src/components/Error/Error.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import cn from 'classnames';
+import { ErrorType } from '../../types/Errors';
+
+type Props = {
+  errorMessage: ErrorType | null,
+  setErrorMessage: (error: ErrorType | null)=> void;
```

Comment:
You are using this `ErrorType | null` in multiple places. Create a `type` for it and import it wherever you need it